README.md

```diff
@@ -114,9 +122,11 @@ journalctl -fu docker.service # Journalctl/Systemd
 
 **VM (Docker-side) Plugin logs**
 
-* Log location: `/var/log/docker-volume-vsphere.log`
-* Config file location: `/etc/docker-volume-vsphere.conf`.
- * This JSON-formatted file controls logs retention, size for rotation
+* Log location (Linux): `/var/log/docker-volume-vsphere.log`
+* Log location (Windows): `C:\Windows\System32\config\systemprofile\AppData\Local\docker-volume-vsphere\logs\docker-volume-vsphere.log`
```

Comment:
Is this the right location for all supported windows versions? Ideally we should put logs somewhere under ProgramData, such as:

C:\ProgramData\VMware\vDVS\Logs\